src/Watcher.ts

```diff
@@ -30,9 +77,12 @@ export default class Watcher {
     this.y = y;
     this.width = width;
     this.height = height;
+    this.min = 0;
+    this.max = 100;
+    this.step = 1;
```

Comment:
I am pressing a key to doubt because `this.step` is already set earlier in this constructor and `min` and `max` ought to be passed through here but aren't (standing issue w/ PR, see #161).

Reply:
Good catch! Removed these.